pkgs/development/tools/misc/openfpgaloader/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, libftdi1
+, libusb
```

```suggestion
, libusb1
```

According to top-level and aliases nix this should be right.